src/lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -138,6 +138,21 @@ export default (accessToken, userID, opts) => {
     ),
     updateMeLoader: gravityLoader("me", {}, { method: "PUT" }),
     orderLoader: gravityLoader(id => `order/${id}`, {}, { headers: true }),
+    confirmOrderLoader: gravityLoader(
```

Comment:
should we rename to `approveOrderLoader` for consistency?

Reply:
Yes, I was confused about what the state name going to be. In some discussion/docs it was `confirm` but in stress and gravity it is `approved`. Are we going with `approve`? cc: @jonallured @ashkan18

Reply:
added some comments about this below :)